sonic_data_client/events_client.go

```diff
@@ -0,0 +1,368 @@
+package client
+
+/*
+#cgo CFLAGS: -g -Wall -I../../sonic-swss-common/common -Wformat -Werror=format-security -fPIE 
+#cgo LDFLAGS: -L/usr/lib -lpthread -lboost_thread -lboost_system -lzmq -lboost_serialization -luuid -lswsscommon
+#include <stdlib.h>
+#include <stdint.h>
+#include <stdbool.h>
+#include "events_wrap.h"
```

Comment:
How do you install this header file?

Reply:
We added support for libswsscommon both in yml file and we reference this headerfile by going to ../../sonic-swss-common/common